src/services/utilities.ts

```diff
@@ -1565,9 +1565,11 @@ namespace ts {
         return n.kind === SyntaxKind.EndOfFileToken ? !!(n as EndOfFileToken).jsDoc : n.getWidth(sourceFile) !== 0;
     }
 
-    export function getNodeModifiers(node: Node): string {
-        const flags = isDeclaration(node) ? getCombinedNodeFlagsAlwaysIncludeJSDoc(node) : ModifierFlags.None;
+    export function getNodeModifiers(node: Node, excludeFlags = ModifierFlags.None): string {

```

Comment:
This shouldn't be done in this change, but it's pretty silly for getNodeModifiers to return a string only for some of its callers to split it again. Since I think this is part of the public API, a good future change would be to add a new function that returns a `ScriptElementKindModifier[]`.

Actually, getSymbolModifiers has the same problem, although none of its callers immediately split its result back into an array.

Reply:
@sandersn You are right. I think we can create an issue for that. I have changes related to `getNodeModifiers`, I tried to use `Set`, in order to about `split`/`join`.